Update and re-order blueprints by stars (#1229)
jhipster/jhipster.github.io: modules/official-blueprints.md

Stream lead is: [Matt Raible](https://github.com/mraible).

## Quarkus

The Quarkus blueprint is supported, in part by the JHipster sponsor [Entando](https://www.entando.com/), and driven by the JHipster community for a [supersonic, subatomic Java back-end](https://quarkus.io/).
Source code and documentation can be found in the [generator-jhipster-quarkus](https://github.com/jhipster/generator-jhipster-quarkus) repository. {⭐️: 111}

Stream lead is [Anthony Viard](https://github.com/avdev4j).

## Micronaut

The Micronaut blueprint is supported and driven by people of the [Micronaut team](https://github.com/jhipster/generator-jhipster-micronaut/graphs/contributors) itself.
It replaces all server side code with a [Micronaut](https://micronaut.io/) application.
Source code and documentation can be found in the [generator-jhipster-micronaut](https://github.com/jhipster/generator-jhipster-micronaut) repository. {⭐️: 91}

Stream lead is [Frederik Hahne](https://github.com/atomfrede).

## Svelte

> The Svelte blueprint replaces the client with Svelete. Source code and documentation could be found in the [generator-jhipster-svelte](https://github.com/jhipster/generator-jhipster-svelte) repository. {⭐️: 44}

Reviewer: ```suggestion
The Svelte blueprint replaces the client side code with SvelteKit framework. Source code and documentation could be found in the [generator-jhipster-svelte](https://github.com/jhipster/generator-jhipster-svelte) repository. {⭐️: 44}
```

# <i class="fa fa-star"></i> Officially supported blueprints

The JHipster team maintains several official blueprints. These blueprints have two main goals:

* Enhance JHipster with new features using different languages and/or support
* Demonstrate how the main generator behavior can be modified to fit anyone's needs

The blueprints below are ordered by the number of stars they have on GitHub. 

## Kotlin

> The Kotlin blueprint replaces most of Java back-end code with Kotlin. Source code and documentation could be found in the [jhipster-kotlin](https://github.com/jhipster/jhipster-kotlin) repository. {⭐️: 411}

Author: ```suggestion
The Kotlin blueprint replaces most of Java back-end code with Kotlin. Source code and documentation could be found in the [jhipster-kotlin](https://github.com/jhipster/jhipster-kotlin) repository. 
{⭐️: 411}
```

# <i class="fa fa-star"></i> Officially supported blueprints

The JHipster team maintains several official blueprints. These blueprints have two main goals:

* Enhance JHipster with new features using different languages and/or support
* Demonstrate how the main generator behavior can be modified to fit anyone's needs

The blueprints below are ordered by the number of stars they have on GitHub. 

## Kotlin

The Kotlin blueprint replaces most of Java back-end code with Kotlin. 

- Code and docs: [jhipster/jhipster-kotlin](https://github.com/jhipster/jhipster-kotlin)
- GitHub ⭐️: 411
- Stream lead: [Sendil Kumar](https://github.com/sendilkumarn)

## .NET Core

The .NET Core blueprint is the first attempt to leave the Java environment and join the .NET world. 

- Code and docs: [jhipster/jhipster-dotnetcore](https://github.com/jhipster/jhipster-dotnetcore)
- GitHub ⭐️: 249

> - Stream lead: [Daniel Petisme](https://github.com/danielpetisme)

Author: ```suggestion
- Stream lead: [Nicolas Raymond](https://github.com/nicolas63)
```

## React Native

The React Native blueprint creates a client application with React Native. 

- Code and docs: [jhipster/generator-jhipster-react-native](https://github.com/jhipster/generator-jhipster-react-native)
- GitHub ⭐️: 235
- Stream lead: [Jon Ruddell](https://github.com/ruddell)

## Node.js

The Node.js blueprint replaces the server java side with the NestJS Node framework. 

- Code and docs: [jhipster/generator-jhipster-nodejs](https://github.com/jhipster/generator-jhipster-nodejs)
- GitHub ⭐️: 223
- Stream lead: [Angelo Manganiello](https://github.com/amanganiello90)

## Ionic

The Ionic blueprint creates a client application with Ionic. 

- Code and docs: [jhipster/generator-jhipster-ionic](https://github.com/jhipster/generator-jhipster-ionic)
- GitHub ⭐️: 171
- Stream lead: [Matt Raible](https://github.com/mraible)

## Quarkus

The Quarkus blueprint is supported, in part by the JHipster sponsor [Entando](https://www.entando.com/), and driven by the JHipster community for a [supersonic, subatomic Java back-end](https://quarkus.io/).

- Code and docs: [jhipster/generator-jhipster-quarkus](https://github.com/jhipster/generator-jhipster-quarkus)
- GitHub ⭐️: 111

> - Stream lead: [Anthony Viard](https://github.com/avdev4j)

Author: ```suggestion
- Stream lead: [Matt Raible](https://github.com/mraible)
```
